package.json

```diff
@@ -37,8 +37,8 @@
 		"ganache": "npx ganache -m 'smart guide what forget tired jungle always expire rescue boring glue champion' --account=\"0x4c3758228f536f7a210f8936182fb5b728046970b8e3215d0b5cb4c4faae8a4e,100000000000000000000\" --account=\"0x34aeb1f338c17e6b440c189655c89fcef148893a24a7f15c0cb666d9cf5eacb3,100000000000000000000\"",
 		"ganache-background": "yarn ganache > /dev/null & client=$! npx wait-port 8545",
 		"geth:build": "docker build ./.github -t gethnode -f .github/Dockerfile",
-		"geth": "docker run -p 8545:8545 gethnode --mine --miner.noverify --datadir data --networkid 4224 --nodiscover --nat \"any\" --ws --ws.addr 0.0.0.0 --ws.port 8545 --http --http.addr 0.0.0.0 --http.port 8545 --allow-insecure-unlock --http.api personal,web3,eth,admin,debug,miner,txpool,net --ws.api personal,web3,eth,admin,debug,miner,txpool,net --unlock 0 --password /pass",
-		"geth-background": "docker run -d -p 8545:8545 gethnode --mine --miner.noverify --datadir data --networkid 4224 --nodiscover --nat \"any\" --ws --ws.addr 0.0.0.0 --ws.port 8545 --http --http.addr 0.0.0.0 --http.port 8545 --allow-insecure-unlock --http.api personal,web3,eth,admin,debug,miner,txpool,net --ws.api personal,web3,eth,admin,debug,miner,txpool,net --unlock 0 --password /pass",
+		"geth": "docker run -p 8545:8545 ethereum/client-go --nodiscover --nousb --ws --ws.addr 0.0.0.0 --ws.port 8545 --http --http.addr 0.0.0.0 --http.port 8545 --allow-insecure-unlock --http.api personal,web3,eth,admin,debug,txpool,net --ws.api personal,web3,eth,admin,debug,miner,txpool,net --dev",
```

Comment:
these commands dont work anymore, should probably remove this and merge and let  @nazarhussain  [PR](https://github.com/ChainSafe/web3.js/pull/4949/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R39) handle this or create a temporary working command.

Reply:
why? now it is works in workflow
`run:  yarn geth-background && yarn test:integration`

Reply:
oh oops not this command, i was running `geth:build` on your branch and this was the error i get 
```
➜  web3.js git:(4809_Add_subscription_Support_to_PromiEvents) ✗ yarn geth:build
yarn run v1.22.10
$ docker build ./.github -t gethnode -f .github/Dockerfile
[+] Building 0.0s (1/2)                                                                                                                                                                                         
 => [internal] load build definition from Dockerfile                                                                                                                                                       0.0s
 => => transferring dockerfile: 2B                                                                                                                                                                         0.0s
failed to solve with frontend dockerfile.v0: failed to read dockerfile: open /var/lib/docker/tmp/buildkit-mount440483704/Dockerfile: no such file or directory
error Command failed with exit code 1.
```

Reply:
yep. removed. thanks!